inspire/base/templates/format/record/Author_HTML_brief.tpl

```diff
@@ -68,7 +68,7 @@
           {% for category in field_categories %}
             {% do category_name.append(category["name"]) %}
           {% endfor %}
-          {{ category_name|join(', ') }}
+          {{ category_name|join_nested_lists }}
```

Comment:
Can you keep it like the `join()` function?

```python
category_name|join_nested_lists(', ')
```
So that one can specify the separator.